main.go

```diff
@@ -35,12 +39,19 @@ env vars: GPTSCRIPT_WORKSPACE_DIR`)
 	case "addElement":
 		tools.AddElement(os.Getenv("DATASETID"), os.Getenv("ELEMENTNAME"), os.Getenv("ELEMENTDESCRIPTION"), []byte(os.Getenv("ELEMENTCONTENT")))
 	case "addElements":
-		var elements []elementInput
-		if err := json.Unmarshal([]byte(os.Getenv("ELEMENTS")), &elements); err != nil {
+		elements, err := handleGzip(os.Getenv("ELEMENTS"))
```

Comment:
```suggestion
		elements := gptscript.GetEnv("ELEMENTS", "")
```

Reply:
Thanks! I didn't know about this function. Kinda surprised I didn't realize its existence while I was searching for `_gz` in the codebase lol.